online-shopping-website/src/App.css

```diff
@@ -24,7 +29,7 @@
 }
 
 .RoutingLink {
-  text-decoration: none;
```

Comment:
Removed `color: white` on merge.

Cannot modify this css label as it is used globally. You'd have to manually override it for the component you need this for.

Reply:
this isn’t present in the latest stuff 😎